spark-jobs/src/main/scala/za/co/absa/enceladus/common/plugin/menas/EventListenerMenas.scala

```diff
@@ -39,6 +45,15 @@ class EventListenerMenas(dao: MenasDAO,
   private var _runUniqueId: Option[String] = None
   private var _runNumber: Option[Int] = None
   private var _runStatus: RunStatus = RunStatus(RunState.running, None)
+  private var _controlMeasure: Option[ControlMeasure] = None
```

Comment:
This `var` seems weird. Is it needed and/or to be used this way? In a way the EventListnerMenas turns into a storage of the currently loaded run. 🤔

Reply:
Yeah. This class here is the most terrible piece of code in Enceladus. But I don't want to refactor it as a part of this PR.

Reply:
OK, I understand